packages/app/src/app/pages/Sandbox/Editor/Header/SandboxName/index.tsx

```diff
@@ -90,66 +81,72 @@ export const SandboxName: FunctionComponent = () => {
       (currentSandbox.team ? currentSandbox.team.name : 'My Sandboxes')
     : 'My Sandboxes';
 
-  const spring = useSpring({
-    opacity: updatingName ? 0 : 1,
-    pointerEvents: updatingName ? 'none' : 'initial',
-  });
   const { customTemplate, owned } = currentSandbox;
 
   return (
     <Main>
-      <Container>
-        {!customTemplate && owned && (
-          <animated.div style={spring}>
-            <Folder>
-              {isLoggedIn ? (
-                <FolderName
-                  onClick={() => modalOpened({ modal: 'moveSandbox' })}
-                >
-                  {folderName}
-                </FolderName>
-              ) : (
-                'Anonymous '
-              )}
-
-              <span role="presentation">/ </span>
-            </Folder>
-          </animated.div>
+      <Stack align="center">
+        {!customTemplate && owned && !updatingName && (
+          <Folder>
+            {isLoggedIn ? (
+              <Button
+                variant="link"
+                css={css({ fontSize: 3, width: 'auto' })}
+                onClick={() => modalOpened({ modal: 'moveSandbox' })}
+              >
+                {folderName}
+              </Button>
+            ) : (
+              'Anonymous '
+            )}
+            <Text role="presentation" variant="muted">
+              /
+            </Text>
+          </Folder>
         )}
 
         {updatingName ? (
-          <Form onSubmit={submitNameChange}>
-            <NameInput
-              autoFocus
-              innerRef={(el: HTMLInputElement) => {
-                if (el) {
-                  el.focus();
-                }
-              }}
-              onBlur={handleBlur}
-              onChange={handleInputUpdate}
-              onKeyUp={handleKeyUp}
-              placeholder={name}
-              value={value}
-              arial-label="sandbox name"
-            />
-          </Form>
+          <>
+            <Form onSubmit={submitNameChange}>
+              <NameInput
+                autoFocus
+                innerRef={(el: HTMLInputElement) => {
+                  if (el) {
+                    el.focus();
+                  }
+                }}
+                onBlur={handleBlur}
+                onChange={handleInputUpdate}
+                onKeyUp={handleKeyUp}
+                placeholder={name}
+                value={value}
+                arial-label="sandbox name"
+              />
+            </Form>
+          </>
         ) : (
-          <Name
-            as={owned ? 'button' : 'span'}
-            onClick={owned ? handleNameClick : noop}
-            owned={owned}
-            aria-label={
-              owned ? `${sandboxName}, change sandbox name` : sandboxName
-            }
-          >
-            {sandboxName}
-          </Name>
+          <>
+            {owned ? (
+              <Button
```

Comment:
Can you please copy over the aria aswell?

Reply:
yep! my bad